sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items -batch-scoring/coco2jsonl.py

```diff
@@ -0,0 +1,127 @@
+import json
```

Comment:
[Minor] There is an extra space in the name of the folder.

Reply:
Done